src/python/pants/backend/jvm/tasks/scalafmt.py

```diff
@@ -5,14 +5,17 @@
 from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
                         unicode_literals, with_statement)
 
+import os.path
```

Comment:
It's not super-important, but for uniformity note that we usually just `import os`. 

But I don't think you need this import at all given my other comment below...